app/views/ui-components/v1/navs/_primary_nav.html.slim

```diff
@@ -19,7 +19,7 @@
                 = render partial: 'exchanges/hbx_profiles/shared/nav_link', locals: {tab_name: 'new-consumer-application', link_path: main_app.begin_consumer_enrollment_exchanges_agents_path(original_application_type: 'phone'), display_name: 'New Consumer Application', active_tab: "new-consumer-application", remote: false, pundit_object: ConsumerRole, pundit_method: :access_new_consumer_application_sub_tab?}
               li
                 = render partial: 'exchanges/hbx_profiles/shared/nav_link', locals: {tab_name: 'identity_verification', link_path: main_app.identity_verification_exchanges_hbx_profiles_path, display_name: 'Identity Verification', active_tab: "identity_verification", remote: false, pundit_object: HbxProfile, pundit_method: :access_identity_verification_sub_tab?}
-              - if EnrollRegistry.feature_enabled?(:aca_individual_market_feature)
+              - if EnrollRegistry.feature_enabled?(:non_aca_individual_market_feature)
```

Comment:
This condition isn't going to work here. The partial needs to be rendered regardless of the feature.